protos/foundation/protos/doc/entity.proto

```diff
@@ -36,16 +38,20 @@ message GenericEntity {
 	// The 'type' field tells a decoder how to deserialize the 'data' field.
 	required string type = 1;
 	required bytes data = 2;
+	required geometry.BBox bbox = 3;
 }
 
-/* Corresponds to an OCR input word. */
-message OcrWord {
-	required ocr.InputWord word = 1;
+/* Corresponds to an OCR word. */
+message Word {
+	required string text = 1;
+	required geometry.BBox bbox = 2;
```

Comment:
Considering message above, I think this is the only message in here that should have BBox instead of BBoxes

Reply:
It is important that every entity type have either just a bbox or a collection of them.

A major reason we are doing this is to simplify assumptions. I would like that tools can assume that (1) a document is a bounding box and a collection of entities, and (2) an entity is a bounding box in that document and then some type-specific data, and eventually some version of (3) an extraction is a field-name-to-entity dictionary.

So I'd really like to enforce that small amount of uniformity.